buildenv/jenkins/jobs/infrastructure/omrMirror

```diff
@@ -34,44 +34,41 @@ pipeline {
             steps {
                 timestamps {
                     script {
-                        withCredentials([usernamePassword(credentialsId: 'b6987280-6402-458f-bdd6-7affc2e360d4', usernameVariable: 'USERNAME', passwordVariable: 'PASSWORD')]) {
-                            if (!fileExists('omr/HEAD')) {
-                                sh "git clone --mirror ${HTTP}${SRC_REPO} omr"
-                            } else {
-                                dir('omr') {
-                                    sh 'git remote update --prune'
-                                }
-                            }
+                        if (!fileExists('omr/HEAD')) {
+                            sh "git clone --mirror ${HTTP}${SRC_REPO} omr"
+                        } else {
                             dir('omr') {
-                                sh "git config remote.target.url ${HTTP}${USERNAME}:${PASSWORD}@${TARGET_REPO}"
-                                sh 'git push target --all'
-                                sh 'git push target --tags'
-                                // Remove the bot account username/password from the config
-                                sh "git config remote.target.url ${HTTP}${TARGET_REPO}"
+                                sh 'git remote update --prune'
+                            }
+                        }
+                        dir('omr') {
+                            withCredentials([usernamePassword(credentialsId: 'b6987280-6402-458f-bdd6-7affc2e360d4', usernameVariable: 'USERNAME', passwordVariable: 'PASSWORD')]) {
+                                sh 'git push ${HTTP}${USERNAME}:${PASSWORD}@${TARGET_REPO} --all'
+                                sh 'git push ${HTTP}${USERNAME}:${PASSWORD}@${TARGET_REPO} --tags'
```

Comment:
My only thought was combining `--all` and `--tags` in one command, but the help for `push` suggests they're mutually exclusive. :-(